Makefile

```diff
@@ -0,0 +1,60 @@
+# The release version.
+VERSION ?= v0.1
+
+# Etcd version. Should match the version in the database/Dockerfile.
+ETCD_VERSION ?= v3.3.7
+
+# Constants
+PYTHON_VERSION ?= 2.7
+
+calico/yaobank-customer:
+	docker build -t $@ customer
+
+calico/yaobank-summary:
+	docker build -t $@ summary
+
+calico/yaobank-database:
+	# Start an etcd server
+	-docker rm -f yaobank-etcd
+	docker run --detach \
+	  --net=host \
+	  --entrypoint=/usr/local/bin/etcd \
+	  --name yaobank-etcd quay.io/coreos/etcd:$(ETCD_VERSION) \
+	  --advertise-client-urls "http://$(LOCAL_IP_ENV):2379,http://127.0.0.1:2379,http://$(LOCAL_IP_ENV):4001,http://127.0.0.1:4001" \
+	  --listen-client-urls "http://0.0.0.0:2379,http://0.0.0.0:4001"
+	# Run the python population script
+	docker run --net=host -v $(CURDIR)/database:/database python:$(PYTHON_VERSION) sh -c "pip install python-etcd && cd /database && python loaddata.py"
+	# Copy out the default.etcd data from the etcd container and delete the etcd container
+	docker cp yaobank-etcd:/default.etcd $(CURDIR)/database
```

Comment:
would be nice to split this recipe up

`database/default.etcd: database/data.txt database/loaddata.py`
with the recipe up to here.

Then a
`calico/yaobank-database: database/default.etcd`
target with the recipe from this comment down.

That make is so that we don't have to rerun the loaddata.py if the data hasn't changed.